src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import core.basesyntax.fileoperations.FileReader;
+import core.basesyntax.fileoperations.FileReaderImpl;
+import core.basesyntax.fileoperations.FileWriter;
+import core.basesyntax.fileoperations.FileWriterImpl;
+import core.basesyntax.fileoperations.ReportGenerator;
+import core.basesyntax.fileoperations.ReportGeneratorImpl;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.OperationStrategy;
+import core.basesyntax.operations.OperationStrategyImpl;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
+        // 1. Read the data from the input CSV file
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.csv");
```

Comment:
Avoid hardcoding file paths in the service classes. Pass the file path as a parameter to your `main` method and use it here. This will make your application more flexible and easier to maintain.